pkg/terminal/colorize/colorize.go

```diff
@@ -284,6 +302,11 @@ func (w *lineWriter) Write(style Style, data []byte, last bool) {
 			}
 			cur = i + 1
 		}
+		if data[i] == '\t' {
```

Comment:
Turn this and the if above into a single switch on `data[i]`. Also since tab is only ever called once you could inline it here.

Reply:
I followed the `nl()` convention, but sure.